### ISSUE TITLE
Question about `mri_convert`

### ISSUE THREAD

**Issue by sravan953** [abhi4ssj/quickNAT_pytorch#25]
Hello,

The readme states:
> Before deploying our model you need to standardize the MRI scans. Use the following command from FreeSurfer
mri_convert --conform <input_volume.nii> <out_volume.nii>
The above command standardizes the alignment for QuickNAT, re-samples to isotrophic resolution (256x256x256) with some contrast enhamcement. It takes about one second per volume.

Assuming FreeSurfer is not available, could you please elaborate on the pre-processing steps?
1. What exactly does it mean to standardize alignment?
2. Is the contrast enhancement necessary?

**Comment by han-yeol** [comment]
Obviously, QuickNAT is implemented with Freesurfer. so I think you must use Freesurfer's mri_convert function.

**Comment by han-yeol** [comment]
Furthermore, This paper is NOT focused on MRI image's preprocessing.
